CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## master / unreleased
 
+## 0.5.0 / 2020-01-15
+
+Note that the ruler flags need to be changed in this upgrade. You're moving from a single node ruler to something that can be sharded.
```

Comment:
This is good, but I suggest to call attention to the fact that you might _need_ to shard.  E.g. if you've set `ruler.num-workers` higher.